### PR TITLE
docs: screenshot #16

### LINKED ISSUE #16: Readme screenshot has cursor in it
I noticed that our screenshot has a cursor in it.

### PR DESCRIPTION
**Pull request by AllanRegush**
closes #16

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 EddieBotUI is the front end application that shows **Eddie Jaoude** Discord Community's public stats.
 
-![EddieBotUI ScreenShot](https://user-images.githubusercontent.com/17693494/90715347-35c78b00-e267-11ea-9c26-c505bc06a5f1.png)
+![EddieBotUI ScreenShot](https://user-images.githubusercontent.com/17693494/91005375-fc0ec100-e593-11ea-957d-b65fb6edb086.png
```
Comment (eddiejaoude) [inline README.md]: Missing the closing bracket

**Comment by eddiejaoude** [comment]
Thank you @AllanRegush , looks good 👍 , just 1 little comment 🤓

File: README.md (hunk 1 above)
Comment (AllanRegush) [reply README.md]: Whoops. Good thing PRs are peer reviewed :facepalm:

**Review by eddiejaoude — APPROVED** [review]
Thank you 👍

**Pull request merged**